.github/scripts/check-status.py

```diff
@@ -1,59 +1,22 @@
 #!/usr/bin/env python3
 
-import re
-from os import environ, path
-from github import Github
-from stdm import get_latest_artifact_url
+from sys import argv
+from pathlib import Path
+from re import compile as re_compile
 
+PACKAGE_RE = re_compile("symbiflow-arch-defs-([a-zA-Z0-9_-]+)-([a-z0-9])")
 
-def check_status():
-    gh_ref = environ['GITHUB_REPOSITORY']
-    gh_sha = environ['INPUT_SHA']
-
-    MAIN_CI = "Architecture Definitions"
-
-    print('Getting status of %s @ %s...' % (gh_ref, gh_sha))
-
-    status = Github(
-        environ['INPUT_TOKEN']
-    ).get_repo(gh_ref).get_commit(sha=gh_sha).get_combined_status()
-
-    for item in status.statuses:
-        print('· %s: %s' % (item.context, item.state))
-
-    if status.state != 'success':
-        print('Status not successful. Skipping...')
-        exit(1)
-
-    if not any([item.context == MAIN_CI for item in status.statuses]):
-        print('Main CI has not completed. Skipping...')
-        exit(1)
-
-    artifacts, _ = get_latest_artifact_url()
-
-    PACKAGE_RE = re.compile("symbiflow-arch-defs-([a-zA-Z0-9_-]+)-([a-z0-9])")
-
-    for artifact in artifacts:
-        name = artifact["name"].split(".")[0]
-        url = artifact["url"]
-
-        m = PACKAGE_RE.match(name)
-        assert m, "Package name not recognized! {}".format(name)
+with (Path(__file__).parent.parent.parent / 'packages.list').open('r') as rptr:
+    for artifact in rptr.read().splitlines():
+        m = PACKAGE_RE.match(artifact)
+        assert m, f"Package name not recognized! {artifact}"
 
         package_name = m.group(1)
-
         if package_name == "install":
-            file_name = "symbiflow-toolchain-latest"
-        elif package_name == "benchmarks":
-            file_name = "symbiflow-benchmarks-latest"
-        else:
-            file_name = "symbiflow-{}-latest".format(package_name)
-
-        with open(path.join("install", file_name), "w") as f:
-            f.write(url)
-
+            package_name == "toolchain"
 
-try:
-    check_status()
-except Exception:
-    print('::set-output name=skip::true')
+        with (Path("install") / f"symbiflow-{package_name}-latest").open("w") as wptr:
+            wptr.write(
+                'https://storage.googleapis.com/symbiflow-arch-defs/artifacts/prod/foss-fpga-tools/'
+                f'symbiflow-arch-defs/continuous/install/{argv[1]}/{artifact}'
+            )
```

Comment:
It might be worth starting to transition to the `f4pga` naming of the packages as well, at least for the `latest` packages.

Reply:
I believe that preserving the old URL for backwards compatibility is a requirement. However, we can push to multiple URLs at the same time. What's the new f4pga URL/naming?

Reply:
Yeah, I think we might go with `f"f4pga-{package_name}-latest"` and push it alongside with the legacy `symbiflow` one than.